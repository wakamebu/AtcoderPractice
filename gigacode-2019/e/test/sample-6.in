4 1000
37 426
725 16 612
237 19 458
516 13 509
408 17 400
