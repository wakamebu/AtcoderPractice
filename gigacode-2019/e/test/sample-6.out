46.861585850556437549130
